server/models/procedures.sql

```diff
@@ -1082,17 +1082,17 @@ BEGIN
 
     IF v_i = 0 OR v_i = fyNumberOfMonths + 1 THEN
       -- Extremum periods 0 and N+1
-      -- Insert periods with null dates
-      INSERT INTO period(`fiscal_year_id`, `number`, `start_date`, `end_date`, `locked`)
-      VALUES (fiscalYearId, v_i, NULL, NULL, 0);
+      -- Insert periods with null dates - period id is YYYY00
+      INSERT INTO period (`id`, `fiscal_year_id`, `number`, `start_date`, `end_date`, `locked`)
+      VALUES (CONCAT(YEAR(fyStartDate), v_i), fiscalYearId, v_i, NULL, NULL, 0);
```

Comment:
What is ```v_i```?

Reply:
It is just an increment variable, like the `i` in `for i in range(10):`.  It could be renamed as needed.  Do you think this should be renamed before merge?

Reply:
If possible yes.

Reply:
Cheers, will do so.